src/main/java/org/assertj/core/api/ComparatorFactory.java

```diff
@@ -12,8 +12,7 @@
  */
 package org.assertj.core.api;
 
-import static java.lang.Math.abs;
-
+import java.math.BigDecimal;
 import java.util.Comparator;
 
 public class ComparatorFactory {
```

Comment:
Can you create two test classes `ComparatorFactory_doubleComparatorWithPrecision_Test` and `ComparatorFactory_floatComparatorWithPrecision_Test`, each should have a parameterized test verifying the behavior of the comparator on different values. You can look at `Offset_Test` for parameterized test examples.

Reply:
OK. Within this week.

Reply:
> Can you create two test classes `ComparatorFactory_doubleComparatorWithPrecision_Test` and `ComparatorFactory_floatComparatorWithPrecision_Test`, each should have a parameterized test verifying the behavior of the comparator on different values. You can look at `Offset_Test` for parameterized test examples.

Can I design a super class named `ComparatorFactory_ComparatorWithPrecision_Test` to avoid the code duplication on these 2 testcases?